mxcubeweb/core/components/user/usermanager.py

```diff
@@ -265,15 +313,24 @@ def _get_configured_roles(self, user):
 
         return list(roles)
 
-    def db_create_user(self, user: str, password: str, lims_data: dict):
+    def db_create_user(self, user: str, password: str, lims_data: dict) -> User:
+        """
+        Create new user in datastore. If the user already exists,
+        update the user information.
+
+        :param str user: representation of (patial) username and nickname of new user
+        :param str password: password
+        :param dict lims_data: dictionary with the lims data to be updated
```

Comment:
If there are type hints, then there is no need to add the type in the docstring as well. Type hint annotation is preferred and enough.

Reply:
ok